wom/src/main/scala/wom/callable/TaskDefinition.scala

```diff
@@ -25,18 +24,19 @@ case class TaskDefinition(name: String,
 
   override lazy val graph: ErrorOr[Graph] = TaskCall.graphFromDefinition(this)
 
-  def lookupFunction(knownInputs: WorkflowCoercedInputs,
-                     wdlFunctions: WdlFunctions[WdlValue],
-                     outputResolver: OutputResolver = NoOutputResolver,
-                     shards: Map[Scatter, Int] = Map.empty[Scatter, Int],
-                     relativeTo: Scope = null): String => WdlValue = ???
+//  def lookupFunction(knownInputs: WorkflowCoercedInputs,
+//                     wdlFunctions: WdlFunctions[WdlValue],
+//                     outputResolver: OutputResolver = NoOutputResolver,
+//                     shards: Map[Scatter, Int] = Map.empty[Scatter, Int],
+//                     relativeTo: Scope = null): String => WdlValue = ???
 
   def instantiateCommand(taskInputs: WomEvaluatedCallInputs,
                          functions: IoFunctionSet,
                          valueMapper: WdlValue => WdlValue = identity[WdlValue],
                          separate: Boolean = false): Try[String] = {
     val mappedInputs = taskInputs.map({case (k, v) => k.localName -> v})
     // TODO: Bring back inputs: Try(StringUtil.normalize(commandTemplate.map(_.instantiate(declarations, taskInputs, functions, valueMapper)).mkString("")))
+    // TODO WOM this is super WDL specific and doesn't belong here.
```

Comment:
Is this actually being used? If it is, then the CWL stuff is presumably crowbarring itself around it? (in which case, it's no longer WDL specific... I guess?)

Reply:
It's used all over: WDL, CWL, WOM.

Reply:
The signature of `instantiateCommand` looks pretty `WOM`y to me, but the `Try(StringUtil.normalize(commandTemplate.map(blablabla` is probably too WDL specific yeah, we'll find out pretty soon when we try to instantiate complex CWL commands